test/EzMap.IntegrationTest/Infrastructure/TestHelper.cs

```diff
@@ -16,4 +16,9 @@ public static async Task<string> GetDefaultUserToken(HttpClient client)
         var token = await tokenResponse.Content.ReadAsStringAsync();
         return token;
     }
+
+    public static string GetErrorMessageOfAProperty(Dictionary<string, string[]> errorDict, string propertyName, int index)
```

Comment:
Try to avoid A, An, The in your naming. This should be `GetPropertyMessage`, or `GetMessage` is enough, because we provide `propertyName` as parameter/key, it should be self-explanatory.

Also should this even be a method no? Feels like it's super straightforward, there are 3 parameters while `errorDict[propertyName][index]` only requires 2.

Reply:
removed